management/position.py

```diff
@@ -0,0 +1,58 @@
+# Converts the required string to its position in the SQLite database.
+# Raises an error if it cannot find the string.
+def positionof(column):
+    if column == "id":
+        return 0
+    if column == "name":
+        return 1
+    if column == "emoji":
+        return 2
+    if column == "activity":
+        return 3
+    if column == "channel":
+        return 4
+    if column == "role":
+        return 5
+    if column == "fakerole":
+        return 6
+    if column == "uses":
+        return 7
+    if column == "votes":
+        return 8
+    if column == "threatened":
+        return 9
+    if column == "enchanted":
+        return 10
+    if column == "demonized":
+        return 11
+    if column == "powdered":
+        return 12
+    if column == "frozen":
+        return 13
+    if column == "undead":
+        return 14
+    if column == "bites":
+        return 15
+    if column == "bitten":
+        return 16
+    if column == "souls":
+        return 17
+    if column == "lovers":
+        return 18
+    if column == "sleepers":
+        return 19
+    if column == "amulets":
+        return 20
+    if column == "zombies":
+        return 21
+    
+    raise SyntaxError("Unable to convert \'{}\' to SQLite position.".format(column))
+
+# This piece is only run when this is the main file in the terminal.
+# It will not run if the file is being imported by another file.
+if __name__ == "__main__":
+
+    print positionof("demonized")
+    print positionof("bites")
+    print positionof("id")
+    print positionof("Something that doesn't exist! :D")
```

Comment:
Probably should move to the tests file

Reply:
We could do that as well. But this one's mainly for quick testing if you're unsure if that specific file works or not.

I am gonna change it, however, as the syntax is Python 2-friendly, but won't be accepted in Python 3.